actions/status-checker/src/pull-updater.ts

```diff
@@ -48,12 +48,10 @@ export async function tryUpdatePullRequestBody(token: string) {
           body: updatedBody
         });
 
-        if (response) {
-          if (response.status === 200) {
-            notice('Pull request updated...');
-          } else {
-            warning('Unable to update pull request...')
-          }
+        if (response && response.status === 200) {
+          notice('Pull request updated...');
+        } else {
```

Comment:
This changes the logic - was that intentional @IEvangelist?

Reply:
Yes it was, I wanted to ensure that all cases were logged appropriately. Before that it didn't log for one case.